index.html

```diff
@@ -614,6 +655,152 @@ <h4>
+          <th>
+            Value type
+          </th>
+          <th>
+          Required
+          </th>
+        </tr>
+        <tr>
+          <td>
+            posture
+          </td>
+          <td>
+            String
+          </td>
+          <td>
+            yes
+          </td>
+        </tr>
+      </table>
+      <p>
+        The [=remote end steps=] are:
+      </p>
+      <ol class="algorithm" data-cite="WEBDRIVER2">
+        <li>Let |posture| be the result of invoking
+        <a data-cite="!WEBDRIVER2#dfn-getting-properties">get a property</a>
+        "posture" from |parameters|.
+        </li>
+        <li>If |posture| is not a [=string=], return [=error=] with
+        [=error code|WebDriver error code=] [=invalid argument=].
+        </li>
+        <li>If |posture| is neither "{{DevicePostureType/continuous}}" nor
+        "{{DevicePostureType/folded}}", return [=error=] with
+        [=error code|WebDriver error code=] [=invalid argument=].
+        </li>
+        <li>Let |topLevelTraversable| be the
+        <a data-cite="webdriver2/#dfn-current-browsing-context">current browsing
+        context</a>'s [=browsing context/top-level traversable=].
+        </li>
+        <li>Set |topLevelTraversable|.<a data-link-for="top-level traversable">
+        [[\PostureOverride]]</a> to |posture|.
+        </li>
+        <li>Let |document| be |topLevelTraversable|'s [=navigable/active
+        document=]
+        </li>
+        <li>Invoke [=device posture change steps=] with |document|.
+        </li>
+        <li>Return [=success=] with data <code>null</code>.
+        </li>
+      </ol>
+      <h4>
+        Clear device posture
+      </h4>
+      <table class="simple">
+        <tr>
+          <th>
+            HTTP Method
+          </th>
+          <th>
+            [=extension command URI Template|URI Template=]
+          </th>
+        </tr>
+        <tr>
+          <td>
+            DELETE
+          </td>
+          <td>
+            /session/{session id}/deviceposture
+          </td>
+        </tr>
+      </table>
+      <p>
+        This [=extension command=] removes device posture override and returns
+        device posture control back to hardware.
+      </p>
+      <p>
+        The [=remote end steps=] are:
+      </p>
+      <ol class="algorithm">
+        <li>Let |topLevelTraversable| be the
+        <a data-cite="webdriver2/#dfn-current-browsing-context">current browsing
+        context</a>'s [=browsing context/top-level traversable=].
+        </li>
+        <li>If |topLevelTraversable|.<a data-link-for="top-level traversable">
+        [[\PostureOverride]]</a> is <code>null</code>, return [=success=] with
+        data <code>null</code>.
+        </li>
+        <li>Set |topLevelTraversable|.<a data-link-for="top-level traversable">
+        [[\PostureOverride]]</a> to
+        <code>null</code>.
+        </li>
+        <li>Let |document| be |topLevelTraversable|'s [=navigable/active
+        document=]
```

Comment:
```suggestion
        document=].
```

Reply:
Remembered in https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/facde85c733cf1b19ad25a6eef6a281eed7e3fa9